### PR TITLE
Add PineryIUS fields for Guppy

### PR DESCRIPTION
**Pull request by callunity**
GR-823 & GLT-3023
Waiting on a Pinery and then pipedev release (due November 11).

### CODE REVIEW

**Comment by callunity** [comment]
Tests will fail until Pinery is released with the relevant changes.

**Comment by apmasell** [comment]
Yes, but the code changes look good.

**Comment by callunity** [comment]
Great, just wanted to make it clear for the second reviewer :)

File: plugin-niassa+pinery/src/main/java/ca/on/oicr/gsi/shesmu/pinery/PineryIUSValue.java [hunk 1]
```diff
@@ -140,6 +152,7 @@ public boolean equals(Object o) {
         && project.equals(that.project)
         && rin == that.rin
         && run_status.equals(that.run_status)
+        && sequencing_kit.equals(that.sequencing_kit)
```
Comment (djcooke) [inline plugin-niassa+pinery/src/main/java/ca/on/oicr/gsi/shesmu/pinery/PineryIUSValue.java]: Could be null

Comment (callunity) [reply plugin-niassa+pinery/src/main/java/ca/on/oicr/gsi/shesmu/pinery/PineryIUSValue.java]: Thanks. I've updated them to be Optionals to account for that.

**Comment by callunity** [comment]
Right now they've got empty string as the default, so nothing will blow up. I don't currently have the capacity to fix it here and update the olives which use these fields, but will try to get it in before the Pinery release on Monday!

**Pull request merged**